hollow-explorer-ui/src/main/resources/explorer-header.vm

```diff
@@ -34,11 +34,14 @@
 			#end
 			#if($stateVersion)
 				<td class="nav">[ State Version: $stateVersion ]</td>
-			#end			
-			#if($type)
+			#end
+			#if($type && !($type.size() > 0))
 				<td class="nav">[ Type: <span style="font-family: monospace;">$esc.html($type)</span> ]</td>
 				<td class="nav">[ <a href="$basePath/type?type=$esc.url($type)">Browse Data</a> | <a href="$basePath/schema?type=$esc.url($type)">Browse Schema</a> ]</td>
 			#end
+			#if(!$type)
```

Comment:
should we be checking that $topLevelTypes is non null?

Reply:
It isn't strictly necessary given we populate the type on every other page, other than the home page. It is a good idea for the future, in-case other pages are added which also don't populate $type. Updated to check `$topLevelTypes`.